index.html

```diff
@@ -0,0 +1,191 @@
+    </header>
+    <p>When you declare a variable outside of any function, it is called a global variable, 
+        because it is available to any other code in the current document. When you declare 
+        a variable within a function, it is called a local variable, because it is available only 
+        within that function.
+    </p>
+    <code>if (true) { var x = 5; } console.log(x); // 5 --Before 2015</code>
+    <code>if (true) { let y = 5; } console.log(y); // ReferenceError: y is
+        not defined -- ECMAScript 2015</code>
+  </section>
+   <section class="main-section" id="Global_variables"> 
+    <header>
+      Global variables
+    </header>
+    <p>Global variables are in fact properties of the global object. In web pages the 
+        global object is window, so you can set and access global variables using 
+        the window.variable syntax.</p>
+
+    <p>Consequently, you can access global variables declared in one window or frame from another window or frame by specifying the window or frame name. For example, if a variable called phoneNumber is declared in a document, 
+        you can refer to this variable from an iframe as parent.phoneNumber.</p>
+  
+  </section>
+   <section class="main-section" id="Constants"> 
+    <header>
+      Constants
+    </header>
+    <p>You can create a read-only, named constant with the const keyword. 
+        The syntax of a constant identifier is the same as for a variable identifier: 
+        it must start with a letter, 
+        underscore or dollar sign and can contain alphabetic, numeric, or underscore characters.</p>
+    <code>const PI = 3.14;</code>
+  </section>
+   <section class="main-section" id="Data_types"> 
+    <header>
+      Data types
+    </header>
+    <p>The latest ECMAScript standard defines seven data types:</p>
+
+     <ul>   
+        
+        <li>Boolean. true and false.</li>
+        <li>null. A special keyword denoting a null value.</li> 
+        <li>Number. 42 or 3.14159.</li>
+        <li>String. "Howdy"</li>
+        <li>Symbol (new in ECMAScript 2015).</li> 
+        <li>Object</li>
+
+     </ul>
+    
+  </section>
+   <section class="main-section" id="if...else_statements"> 
+    <header>
+      if...else Statements
+    </header>
+    <p>Use the if statement to execute a statement if a logical 
+        condition is true. Use the optional else clause to execute 
+        a statement if the condition is false. An if statement looks as follows:
+    </p>
+    <code>if (condition) { statement_1; } else { statement_2; }</code>   
+    <code>if (condition_1) { statement_1; } else if (condition_2) {
+        statement_2; } else if (condition_n) { statement_n; } else {
+        statement_last; }</code>
+    <code>if (x = y) { /* statements here */ }</code>
+    <code>if ((x = y)) { /* statements here */ }</code>
+  </section>
+   <section class="main-section" id="while_statements"> 
+    <header>
+      while statements
+    </header>
```

Comment:
✔️ Good job done using Ids. But it seems your code has so many ids. 🤔 I think just one or two is okay for a page. Many ids can create problems for managing styles in one place
id and header issues

✔️ Good semantics job by using header. But some headers seems to have no style class. 🤔 I think If you have others pages where headers change it will be difficult to manage. Kindly work on it.
id and header issues2